logstash-core/src/main/java/org/logstash/execution/PipelineReporterExt.java

```diff
@@ -81,9 +81,9 @@ public final class PipelineReporterExt extends RubyBasicObject {
     private static final RubyString DEAD_STATUS =
         RubyUtil.RUBY.newString("dead").newFrozen();
 
-    private IRubyObject logger;
+    private transient IRubyObject logger;
 
-    private IRubyObject pipeline;
+    private transient IRubyObject pipeline;
```

Comment:
Are we sure that we are not serializing/deserializing reported fields? This class seems to me somewhere sending/saving/reporting data.

Reply:
This class is used to generate a report, both a `RubyHash` with `toHash` method or with `snapshot` to generate an instance of `PipelineReporterExt.SnapshotExt`.
However `PipelineReporterExt` and `SnapshotExt` are never serialized using Java serialization mechanism, because both have `serialVersionUID = 1L`.
They are used to create a json, but that uses another serialization mechanism, and they never deserialized.